backend/src/seed.js

```diff
@@ -1,26 +1,30 @@
 import { UserRepository } from './repository/UserRepository';
-import { TicketRepository } from './repository/TicketTypesRepository';
-let userRepo = new UserRepository()
-let ticketRepo = new TicketRepository()
+import { TicketTypeRepository } from './repository/TicketTypesRepository';
+let userRepo = new UserRepository();
+let ticketTypeRepo = new TicketTypeRepository();
 
-let userDummyData = {
+userRepo.save({
   name: 'Lehel',
   email: 'lehel@gmail.com',
-  password: 'password123',
-};
-let adminDummyData = {
+  password: '$2b$10$H7mQGKBAO66DkB2CY49LzujvzHeO9mB3OheHwzDvKfG7RlgTOq4Vm',
+});
+userRepo.saveAdmin({
```

Comment:
@LLaszlo89 ugh, I've missed this :)
we should have only one method for saving the user, the difference is the user object (that can be admin or not admin shouldn't matter)